src/views/collections/framework-field-guide/segments/quotes.astro

```diff
@@ -213,7 +213,7 @@ New software developers often don't see how programming languages share many com
 		<QuoteCard
 			personName={`Xeno`}
 			personLink="https://www.xenophorium.dev/"
-			avatarSrc={import("../../../../../public/content/data/xeno.jpg")}
+			avatarSrc={import("../people-pictures/xeno.jpg")}
```

Comment:
Every other profile image used in FFG is copied, which I would prefer because that means it doesn't have hidden dependencies around the repo.

If you really want to change that, could we have it resolve from `utils/api` / `getUnicornById` instead? That way it doesn't cause failures if updated in the future (e.g. these could be changed through the CMS).
(IMO this makes more sense to be static/hardcoded as it is now, though.)

Reply:
`getUnicornById` is fine with me - let's go that route